ws-nextjs-app/cypress/e2e/languagesPage/tests.ts

```diff
@@ -0,0 +1,81 @@
+export const homePageTestSuites = [
+  {
+    path: '/ws/languages',
+    service: 'pidgin',
+    runforEnv: ['test'],
+    tests: [
+      ({ service, pageType }: { service: string; pageType: string }) => {
+        describe(`Tests for ${pageType} page - ${service}`, () => {
```

Comment:
I think we can hardcode this (and remove the parameters above)
```suggestion
        describe(`WS Languages page`, () => {
```